tests/Unit/PropertyInfo/Extractor/EpoetryReflectionExtractorTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace OpenEuropa\EPoetry\Tests\PropertyInfo\Extractor;
+
+use OpenEuropa\EPoetry\PropertyInfo\Extractor\MappingReflectionExtractor;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Yaml\Yaml;
+
+/**
+ * Class EpoetryReflectionExtractorTest.
+ *
+ * @internal
+ * @coversNothing
+ */
+final class EpoetryReflectionExtractorTest extends TestCase
+{
+    /**
+     * Data provider.
+     *
+     * @return array
+     */
+    public function getTypesCases()
+    {
+        return $this->getFixture(__DIR__ . '/../../../fixtures/epoetry-reflection-extractor.yml');
```

Comment:
We can remove the `epoetry-` from the file name, I guess this is a leftover from when ePoetry was part of the class name, the extractor is called `MappingReflectionExtractor` so we should reflect this here.

Also I usually map the class namespace in the fixture directories, so this could be `property-info\Extractor\mapping-reflection-extractor.yml`

Reply:
Fixed.